server/api/game/controller.js

```diff
@@ -94,22 +161,48 @@ const moveGhosts = ({ ghosts, gridState, scatterStart }) => {
 
 const calculateNextGameState = () => {
   const {
-    ghosts, gridState, scatterStart,
+    ghosts, gridState, scatterStart, pacmanOne, pacmanTwo,
   } = gameState;
-  const { ghostsUpdated, moveGhostsCount } = moveGhosts(
-    { ghosts, gridState, scatterStart },
-  );
+  const { ghostsUpdated, moveGhostsCount } = moveGhosts({ ghosts, gridState, scatterStart });
+  const { pacmanUpdated: newPacmanOne } = movePacman({
```

Comment:
You are renaming 'pacmanUpdated' to 'newPacmanOne'. Instead rename it to 'updatedPacmanOne'

Reply:
This is temporary, I just moved the client logic to server. Will be removed in refactoring.